Ex5part2.txt

```diff
@@ -0,0 +1,11 @@
+#Usage bash Ex5part2.txt wages.csv
```

Comment:
1.	Your output file (wages3.csv) isn’t quite what we’re looking for. Looking for a file delimited by spaces with only two columns (gender, yearsExperience). File should be sorted by gender and then within each gender by yearsExperience. Here’s some code that would make that happen. cut -d , -f 1,2 wages.csv | grep -v "gender" | sort -t , -k1,1 -k2,2n | uniq | tr ',' ' '
a.	0.6
2.	Code works but doesn’t give the right answers for all the parts. Parts 2,3 are good. Part 1 is incorrect. This would work:  cat wages.csv | grep -v "gender" | sort -t , -k 4 -n –r | head -n 1 | cut -d , -f 1,2,4  # male	5	11	39.80892
a.	0.9
3.	Missing answer for this question

1.5/3